prow/pod-utils/decorate/podspec_test.go

```diff
@@ -454,19 +632,22 @@ func TestProwJobToPod(t *testing.T) {
 								{Name: "ENTRYPOINT_OPTIONS", Value: `{"args":["/bin/thing","some","args"],"timeout":7200000000000,"grace_period":10000000000,"artifact_dir":"/logs/artifacts","process_log":"/logs/process-log.txt","marker_file":"/logs/marker-file.txt"}`},
 								{Name: "GOPATH", Value: "/home/prow/go"},
 								{Name: "JOB_NAME", Value: "job-name"},
-								{Name: "JOB_SPEC", Value: `{"type":"periodic","job":"job-name","buildid":"blabla","prowjobid":"pod","refs":{}}`},
-								{Name: "JOB_TYPE", Value: "periodic"},
+								{Name: "JOB_SPEC", Value: `{"type":"presubmit","job":"job-name","buildid":"blabla","prowjobid":"pod","refs":{"org":"org-name","repo":"repo-name","base_ref":"base-ref","base_sha":"base-sha","pulls":[{"number":1,"author":"author-name","sha":"pull-sha"}],"path_alias":"somewhere/else"}}`},
```

Comment:
Should we expose the fact that we skipped cloning in `JOB_SPEC` somehow? Currently it is possible to determine what code is checked out by examining the refs in `JOB_SPEC`, but this change will violate that.

Reply:
Yeah, we can propagate the value to that as well +1

Reply:
On second thought -- I am not sure we need to. Asking for no cloning in the decorated case is an explicit user action on the job config. I can't really think of a job that should be determining if a clone happened at runtime or not. If a job author is expecting no clone and expresses that intent at configuration time, their code should just do it's own thing and not try to determine if we did or did not clone.